### PR TITLE
Manage ex gratia business reps

### PR DESCRIPTION
**Pull request by CaseyProsser**

### CODE REVIEW

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java [hunk 1]
```diff
@@ -677,6 +636,310 @@ public void theManagementPageShouldBeDisplayed(String pageTitle) {
...
+                muiDashboard.selectDashboardLinkWithText("Manage MPAM campaigns");
+                break;
+            case "TREAT OFFICIAL":
+                muiDashboard.selectDashboardLinkWithText("Manage Treat Official campaigns");
+                break;
+            default:
+                pendingStep(caseType + " is not defined within " + getMethodName());
+        }
+        listsManagement.clickTheAddNewCampaignButton();
+        listsManagement. enterCampaignName();
```
Comment (CPage-Ten10) [inline src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java:732]: Remove the space after the full stop

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java [hunk 2]
```diff
@@ -677,6 +636,310 @@ public void theManagementPageShouldBeDisplayed(String pageTitle) {
...
```
Comment (CPage-Ten10) [inline src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java:766]: Same here

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java [hunk 3]
```diff
@@ -677,6 +636,310 @@ public void theManagementPageShouldBeDisplayed(String pageTitle) {
...
+        listsManagement.assertVisibilityOfBusinessUnitInBusinessUnitTable();
+    }
+
+    @And("I have an existing Business Unit I want to amend")
+    public void iHaveAnExistingBusinessUnitIWantToAmend() {
+        iSelectAManagementUIDashboardLink("Manage MPAM Business Units");
+        listsManagement.selectABusinessArea();
+        clickTheButton("Submit");
+        listsManagement.clickTheAddNewBusinessUnitButton();
+        listsManagement. enterBusinessUnitName();
```
Comment (CPage-Ten10) [inline src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java:845]: Same here

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java [hunk 4]
```diff
@@ -677,6 +636,310 @@ public void theManagementPageShouldBeDisplayed(String pageTitle) {
...
+        listsManagement.assertVisibilityOfEnquiryReasonInEnquiryReasonTable();
+    }
+
+    @And("I have an existing Enquiry Reason I want to amend")
+    public void iHaveAnExistingEnquiryReasonIWantToAmend() {
+        iSelectAManagementUIDashboardLink("Manage MPAM Enquiry Reasons");
+        listsManagement.selectAnEnquirySubject();
+        clickTheButton("Submit");
+        listsManagement.clickTheAddNewEnquiryReasonButton();
+        listsManagement. enterEnquiryReasonName();
```
Comment (CPage-Ten10) [inline src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java:907]: Same here

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java [hunk 5]
```diff
@@ -677,6 +636,310 @@ public void theManagementPageShouldBeDisplayed(String pageTitle) {
...
+        listsManagement.assertSuccessMessageForAmendingEnquiryReasonVisible();
+    }
+
+    @And("I have added a new Enquiry Reason in MUI")
+    public void iHaveAddedANewEnquiryReasonInMUI() {
+        iSelectAManagementUIDashboardLink("Manage MPAM Enquiry Reasons");
+        listsManagement.selectAnEnquirySubject();
+        clickTheButton("Submit");
+        listsManagement.clickTheAddNewEnquiryReasonButton();
+        listsManagement. enterEnquiryReasonName();
```
Comment (CPage-Ten10) [inline src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java:939]: Same here

**Review by CPage-Ten10 — APPROVED** [review]

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java (hunk 1 above)
Comment (CaseyProsser) [reply src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java]: removed

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java (hunk 3 above)
Comment (CaseyProsser) [reply src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java]: removed

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java (hunk 4 above)
Comment (CaseyProsser) [reply src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java]: removed

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java (hunk 5 above)
Comment (CaseyProsser) [reply src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java]: removed

File: src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java (hunk 2 above)
Comment (CaseyProsser) [reply src/test/java/com/hocs/test/glue/decs/ManagementUIStepDefs.java]: removed

**Pull request merged**